docs/backends.rst

```diff
@@ -337,16 +337,25 @@ Create a Provider Class
 
 .. code-block:: python
 
-    from payments.providers.base import BaseProvider
+    from payments.core import BasicProvider
+
+    class MyPaymentProvider(BasicProvider):
+      def get_hidden_fields(self, payment):
+        # Implement custom logic here to generate hidden fields for the payment form
+        hidden_fields = {
+            'custom_param': self.custom_param,
+            # Add more fields as needed
+        }
+        return hidden_fields
 
-    class MyPaymentProvider(BaseProvider):
       def process_data(self, payment, request):
           # Implement webhook processing logic
           pass
 
-      def get_form(self, payment, data=None):
-          # Implement payment form rendering logic
-          pass
```

Comment:
This field is relevant when implementing a provider; some providers require additional non-hidden fields.